org/org.yaml

```diff
@@ -417,38 +417,27 @@ orgs:
         maintainers:
         - bobcatfish
         members:
-        - savitaashture
```

Comment:
thanks @lbernick for spending so much effort on this 🙏 This will help the review process.

@afrittoli @dibyom I had a question, I am a little confused how this list is composed. I am assuming this only considers pipeline repo. Also, no personal offense to any one here but 

https://github.com/tektoncd/pipeline/pulls?q=is%3Apr+reviewed-by%3Asavitaashture yields 0 in 2022 and `savitaashture` is moved alumni which is reasonable.
 
But https://github.com/tektoncd/pipeline/pulls?q=is%3Apr+reviewed-by%3Asthaha yeilds 0 reviews in 2022 but `sthaha` is not moved 🤔 

I just checked these two GitHub profiles before commenting here.

Reply:
Am I making any mistake in interpreting these?

Reply:
Sunil (sthaha) is definitely not active anymore and could be moved to alumni as well 👼🏼

Reply:
Thanks Priti and Vincent! I just moved the people with < 5 reviews ever and 0 reviews in 2022. If we remove the first criterion and just filter to those with 0 reviews in 2022, that would include a few more people. Happy to move those folks to alumni as well if you'd like.

Reply:
Thanks @lbernick, I am open to suggestions, no strong opinion either ways. Do you mind sharing the list of those few more people and check with @tektoncd/core-collaborators? Thanks! Please feel free to leave it if you feel like.

Reply:
Thanks Priti, done in https://github.com/tektoncd/community/pull/973